app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::ReviewsController < ApplicationController
+    def create
+        review = Review.new(review_params)
+        review.user = current_user
+        if review.save
+            render json: review
+        else
+            flash[:errors] = review.errors.full_messages.to_sentence
```

Comment:
flash does not work in API controllers. remove this, and return a message as json if you wish instead